.github/workflows/community-release.yml

```diff
@@ -6,7 +6,26 @@ on:
         required: true
         description: 'Needs to match, exactly, the name of a milestone. The version to be released please respect: major.minor.patch, major.minor.patch-preview or major.minor.patch-preview<number> format. example: 7.4.3, 7.4.3-preview or 7.4.3-preview1'
 jobs:
+  config:
```

Comment:
I think we can revert this - this change only runs manually via `workflow_dispatch`, and the less actions yaml to maintain the better